docs/modules/components/pages/processors/ollama_embeddings.adoc

```diff
@@ -0,0 +1,113 @@
+= ollama_embeddings
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+
+
+////
+     THIS FILE IS AUTOGENERATED!
+
+     To make changes, edit the corresponding source file under:
+
+     https://github.com/redpanda-data/connect/tree/main/internal/impl/<provider>.
+
+     And:
+
+     https://github.com/redpanda-data/connect/tree/main/cmd/tools/docs_gen/templates/plugin.adoc.tmpl
+////
+
+
+component_type_dropdown::[]
+
+
+Processor that uses the Ollama API to create vector embeddings.
```

Comment:
```suggestion
Sends text to your chosen Ollama large language model (LLM) and creates vector embeddings, using the Ollama API. Vector embeddings are long arrays of numbers that represent values or objects, in this case text. 
```
I've suggested a more detailed description using a similar format to the chat processor.